### ISSUE TITLE
Branding: Tagline (Data Done Right) Lockup

### ISSUE THREAD

**Issue by sabrinang** [MozillaFoundation/mozfest-design-2018#25]
### Description:
Design a lockup treatment of the tagline this year which is `Data Done Right` 

**Format**
We can have a main outline version and fill version. Include various alternative examples of modified versions.

### Previous Applications
2016: variations of 'One Web'
2017: variations of 'Web for All'
<img width="953" alt="image" src="https://user-images.githubusercontent.com/2492510/43217960-e5824fcc-8ff7-11e8-846a-18e1ca9b5259.png">

### Reference from Last Year: 
Google Drive Files: https://drive.google.com/open?id=0B4Q8pzCpDS_edjZPTDlMN0RLUFE
Github Issue: https://github.com/MozillaFoundation/mozfest-design-2017/issues/48

**Comment by sabrinang** [comment]
Here is the localisation ticket of the tagline: https://github.com/mozilla/mozillafestival.org/issues/923 

Link to translations for reference here: https://l10n.mozilla-community.org/langchecker/?website=6&file=other/2018/mozfest.lang&action=translate&show#string2

**Comment by sabrinang** [comment]
For this flexible modular lockup system, the alignment can be customized for various designs and can work for other languages as well. HI-IN may need further refinement but can be revisited when l10n tagline posters are iterated on (the only application of l10n tagline we used last year). Tagline lockup margins are created and documented along side size usages. 

Similarly to last year, we iterated and included tagline variants that we can apply colour, gradient and art textures on to produce various design assets.

Iteration and explorations seen in moodboard/WIP slides: https://docs.google.com/presentation/d/1WJ5c6LtdXmB4tkcJO0hy94L2ZVROIdYPGyZwEiqei9U/edit#slide=id.g409af93c24_1_8 and redpen: https://redpen.io/p/vw4f94fde13e0f0b57

Google Drive folder and exports: https://drive.google.com/open?id=1fK-REXx9vEUAvfj82yeFLdzvq3TNIT9t

![mozilla-festival-2018_tagline-lockup-overview](https://user-images.githubusercontent.com/2492510/45119942-a4aa8880-b111-11e8-9816-714c05a30114.jpg)
